src/python/pants/reporting/reporting.py

```diff
@@ -131,12 +140,14 @@ def update_reporting(self, global_options, is_quiet_task, run_tracker):
       safe_mkdir(global_options.logdir)
       run_id = run_tracker.run_info.get_info('id')
       outfile = open(os.path.join(global_options.logdir, '{}.log'.format(run_id)), 'w')
-      settings = PlainTextReporter.Settings(log_level=log_level, outfile=outfile, color=False,
-                                            indent=True, timing=True, cache_stats=True,
+      errfile = open(os.path.join(global_options.logdir, '{}.err.log'.format(run_id)), 'w')
+      settings = PlainTextReporter.Settings(log_level=log_level, outfile=outfile, errfile=errfile,
+                                            color=False, indent=True, timing=True, cache_stats=True,
```

Comment:
would it make more sense to interleave (e.g. `errfile=outfile`) in the case of writing to logs vs stdio?

Reply:
Yea, probably.